src/main/java/com/nylas/Neural.java

```diff
@@ -0,0 +1,82 @@
+package com.nylas;
+
+import okhttp3.HttpUrl;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Neural {
```

Comment:
I didn't include any of the `/feedback` endpoints because I don't think it makes sense to have it in the SDK. Apart from categorize (which is supported), they don't do anything functional that would effect the state of the remaining Neural objects.

Reply:
From what I can tell, the feedback endpoints are for telling Nylas which items had poor results.  I can see a case for a customer wanting to do that programmatically, so I think it makes sense to include in the API.  However, that may be a better conversation to have with Hazik or product folks.  I wouldn't hold up this PR for it.